pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -1088,3 +1165,5 @@ func validateFieldRef(fieldRef string, fldPath *field.Path) field.ErrorList {
 	// no further checks, for an easier upgrade/rollback
 	return nil
 }
+
+var compiler = plugincel.NewCompiler(environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()))
```

Comment:
doesn't need to change now, but if we do ever make CompatibilityVersion selectable in the future, this won't be able to be a package var in this way